android/src/main/java/com/swmansion/reanimated/layoutReanimation/AnimationsManager.java

```diff
@@ -347,6 +348,14 @@ public void setNewProps(
       props.remove(Snapshot.TRANSFORM_MATRIX);
     }
 
+    if (props.containsKey(Snapshot.BORDER_RADIUS)) {
+      Double borderRadius = (Double) props.get(Snapshot.BORDER_RADIUS);
+      if (borderRadius != null) {
+        ((ReactViewBackgroundDrawable)view.getBackground()).setRadius(borderRadius.floatValue());
+      }
+      props.remove(Snapshot.TRANSFORM_MATRIX);
```

Comment:
Shouldn't this be
```suggestion
      props.remove(Snapshot.BORDER_RADIUS);
```
?